cmd/tools/grafana/grafana.go

```diff
@@ -1165,7 +1144,7 @@ func addImportExportFlags(commands ...*cobra.Command) {
 		cmd.PersistentFlags().StringVarP(&opts.addr, "addr", "a", "http://127.0.0.1:3000", "Address of Grafana server (IP, FQDN or hostname)")
 		cmd.PersistentFlags().StringVarP(&opts.token, "token", "t", "", "API token issued by Grafana server for authentication")
 		cmd.PersistentFlags().BoolVarP(&opts.useHTTPS, "https", "S", false, "Use HTTPS")
-		cmd.PersistentFlags().BoolVarP(&opts.overwrite, "overwrite", "o", false, "Overwrite existing dashboard with same title")
+		cmd.PersistentFlags().BoolVarP(&opts.overwrite, "overwrite", "o", true, "Overwrite existing dashboard with same title")
```

Comment:
We should leave this argument with a default value of false. When the arg is set, we should log a warning saying something like: 'the overwrite flag is no longer used and will be removed in a future release. Please remove this flag from your command line invocation. When importing, dashboards are always overwritten.'

Reply:
Added the check.